Police-Data-Accessibility-Project/design-system
# feature(tailwind): Use tailwindcss

Resolves #9, resolves #4

- [x] Install
- [x] Build and pre-publish architecture
- [x] Add CLI for easy copying of assets
- [x] Implement utility classes within current CSS
- [x] Update `demo.html` to target built CSS
- [x] Update `demo.html` for better accessibility (nav) 
- [x] Bump version to `"1.1.0"`
- [x] Update documentation (`README` and `CONTRIBUTING`)


## To test
1. Run `npm run build` Open `demo.html` with live server, compare with [pdap.io](https://pdap.io), note any differences in styling
2. Run `npm link`
3. In another local repository (`pdap.io` or `data-sources-app` would work), run `npm link pdap-design-system` and `npm run build`
4. You should now have a symlink to the local package (since this update has not yet been published).
5. From the root directory of the installing project, run `pdap-design-system --copy-assets --to=test-directory` and  ensure that the assets are copied over to a dir called `/test-directory` in the root of the project.
6. Revert changes from steps 3-5 (These styles will be implemented in other projects once this update is published).
